javatests/arcs/showcase/imports/recipes/BUILD

```diff
@@ -0,0 +1,15 @@
+load(
+    "//third_party/java/arcs/build_defs:build_defs.bzl",
+    "arcs_kt_gen",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+arcs_kt_gen(
+    name = "imports_recipe",
+    srcs = ["imports.arcs"],
```

Comment:
Minor nit: how about naming this target (and file) after the GatherImports recipe that it defines? Or alternatively just "recipes" since that's the package name.

Reply:
I'd rather not name it `recipes`, since I expect there to be more recipes in this file later on. 
I can call it gather-imports.